config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # Non Sufia generated routes
   mount ProductionCredits::Engine, at: "/production_credits"
+  mount RailsAdmin::Engine => '/production_credits/admin', as: 'rails_admin'
```

Comment:
If we're moving RailsAdmin into the ProductionCredits engine, shouldn't that engine be responsible for mounting RailsAdmin instead?  Not sure if that makes sense or is even possible.